src/ResourceManager/Storage/Stack/Commands.Management.Storage/Commands.Management.Storage.csproj

```diff
@@ -52,10 +52,6 @@
       <HintPath>..\..\..\..\packages\Microsoft.Azure.KeyVault.Core.1.0.0\lib\net40\Microsoft.Azure.KeyVault.Core.dll</HintPath>
       <Private>True</Private>
     </Reference>
-    <!--    <Reference Include="Microsoft.Azure.Management.Storage, Version=6.0.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35, processorArchitecture=MSIL">
```

Comment:
lest keep the commented out SDK reference, so that we could easily remember what we copied here

Reply:
We didn't copy anything, this was never released remember?

Reply:
What do you mean?  We have a "Generated" directory.  Are you saying we did release this specific C# SDK for storage?